gtsam/gtsam.i

```diff
@@ -2847,6 +2847,7 @@ virtual class EssentialMatrixFactor : gtsam::NoiseModelFactor {
 
 #include <gtsam/slam/dataset.h>
 class SfmTrack {
```

Comment:
I’d prefer the name `point3`

Reply:
sure, so rename `p` to be `point3`? I think we will need to rename it in the following examples in this PR then since it's a breaking change:
- `examples/CreateSFMExampleData.cpp`
- `timing/timeSFMBALnavTcam.cpp`
- `timing/timeSFMBAL.cpp`
- `examples/SFMExample_bal_COLAMD_METIS.cpp`
- `examples/SFMExampleExpressions_bal.cpp`
- `examples/SFMExample_bal.cpp`

Reply:
Nope, rename `get3dPoint` to `point3`.

Reply:
Ah I see, @dellaert do you mind confirming that and I can quickly make the change? I guess “get*()” is not preferred in GTSAM?

Reply:
Indeed, no get...

Reply:
pushed the update, should be good to go now